source/index.md

```diff
@@ -17,3 +18,4 @@
 - [Server Selection Test Plan](server-selection/server-selection-tests.md)
 - [Unified Test Format](unified-test-format/unified-test-format.md)
 - [Wire Compression in Drivers](compression/OP_COMPRESSED.md)
+- [pytest cache directory #](client-side-encryption/.pytest_cache/README.md)
```

Comment:
Why is this here if `.pytest_cache` is in `.gitignore`?

Reply:
It was there for me locally when I ran the script.  I'll re-run it.